MojioSDK/Client/Images/ImagesClient.swift

```diff
@@ -108,8 +119,10 @@ open class ImagesClient: RestClient {
         }
     }
     
-    open func runImage(completion: @escaping (UIImage?) -> Void, failure: @escaping (Any?) -> Void) {
-        
+    open func runImage(debug: ((_ request: Request?, _ response: DataResponse<UIImage>, _ date: Date?) -> Void)? = nil,
```

Comment:
@andriy-morgunov-epam  I guess "response" should be optional similar to other closures

Reply:
I was wondering about that too but the response for this one is not optional (presumably because it's a UIImage type)